src/Compilers/Core/Portable/InternalUtilities/FatalError.cs

```diff
@@ -236,6 +232,12 @@ public static bool ReportWithoutCrashUnlessCanceledAndPropagate(Exception except
                 Report(exception, s_nonFatalHandler);
             }
 
+            if (exception is OperationCanceledException)
+            {
+                // Wrap in InvalidOperationException since cancellation was not requested
+                throw new InvalidOperationException(exception.Message, exception);
+            }
```

Comment:
📝 This should be removed for release. It's only used by OOP things, but it could cause a `ReportUnlessCancelled` to fail when it would otherwise continue.